### ISSUE TITLE
does user need to have chrome installed or can it be generalized to any default browser?

### ISSUE THREAD

**Issue by prabhakar267** [amanmj/TwitterRefresh#1]
As per this link (https://docs.python.org/2/library/webbrowser.html), I don't think it's necessary to mention the browser. 
```webbrowser``` should open your default browser

**Comment by amanmj** [comment]
Yeah that was the problem on mac on specifying no browser, it should have opened the default browser, but was opening on firefox so had to force chrome.

**Comment by prabhakar267** [comment]
@amanmj was there any problem in firefox getting opened?

**Comment by amanmj** [comment]
@prabhakar267 if a user opens http://localhost:5000/ in chrome and the next tab opens in firefox?
That won't be convenient

**Comment by prabhakar267** [comment]
I think the browser can be detected in ```flask``` from ```User-agents``` (https://stackoverflow.com/questions/9878020/how-do-i-get-the-user-agent-with-flask)
Also, ```webbrowser``` must have some method of opening the current browser or something like that.

**Comment by amanmj** [comment]
Yeah will have to add snippet to detect browser as well OS and open the same browser through ```webbrowser```
